python/ray/scripts/scripts.py

```diff
@@ -1926,6 +1927,93 @@ def cpp(show_library_path, generate_bazel_project_template_to, log_style,
                 " && bash run.sh"))
 
 
+@cli.group(hidden=True)
+@click.option(
+    "--address",
+    type=str,
+    default=None,
+    required=False,
+)
+@click.pass_context
+def job(ctx, address: Optional[str]):
+    if address is None:
+        if "RAY_ADDRESS" not in os.environ:
+            raise ValueError(
+                "Address must be specified using either the --address flag "
+                "or RAY_ADDRESS environment variable.")
+        address = os.environ["RAY_ADDRESS"]
+
+    ctx.obj = JobSubmissionClient(address)
+
+
+@job.command("submit")
+@click.option(
+    "--job-id",
+    type=str,
+    default=None,
+    required=False,
+)
+@click.option(
+    "--address",
+    type=str,
+    default=None,
+    required=False,
+)
+@click.option(
+    "--working-dir",
+    type=str,
+    default=None,
+    required=False,
+)
+@click.argument("entrypoint", nargs=-1, required=True)
+@click.pass_obj
+def job_submit(client: JobSubmissionClient, job_id: Optional[str],
```

Comment:
Could you also add help documentation for all the commands and arguments?

Reply:
Yup will do